dotnetv3/S3/scenarios/TransferUtilityBasics/TransferUtilityBasics/TransferBasics.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[S3.dotnetv3.TransferUtilityBasics.Main]
+
+// The Amazon S3 client here uses the default user credentials
+// defined for this computer.
+IAmazonS3 client = new AmazonS3Client();
+var transferUtil = new TransferUtility(client);
+
+// Change the following values to an Amazon S3 bucket that
+// exists in your Amazon account.
```

Comment:
// Change the following values to an S3 bucket that
// exists in your AWS account.